screens/Home/MyVcs/GetIdInputModal.tsx

```diff
@@ -38,11 +39,39 @@ export const GetIdInputModal: React.FC<GetIdInputModalProps> = (props) => {
                 value={controller.id}
                 keyboardType="number-pad"
                 rightIcon={
-                  controller.isInvalid ? (
-                    <Icon name="error" size={18} color="red" />
-                  ) : (
-                    <Icon name="help" size={18} />
-                  )
+                  <Tooltip
+                    popover={
+                      <Text>
+                        {t('qstnMarkToolTip', {
+                          vcLabel: controller.vcLabel.singular,
+                        })}
+                      </Text>
+                    }
+                    width={Dimensions.get('screen').width * 0.8}
```

Comment:
This is not the correct way. Because 20% is large in a bigger screen while its good in smaller screen.  Is there no other way to make it responsive?